releasenotes/notes/1.3/prepare-1.3.0-7c45598775fc2bbf.yaml

```
@@ -0,0 +1,22 @@
+---
+prelude: >
+    Qiskit version 1.3.0 brings in major performance and quality improvements
+    for the transpiler. The minimum required Python version is now 3.9.
+
+    There have been many new features, fixes and improvements introduced in this
+    new version of Qiskit, some of which include:
+
+      * The majority of the transpiler passes used by the preset pass 
+        manager have been ported into Rust, resulting in a 6x overall runtime 
+        improvement compared to Qiskit 1.2.4. The ported passes include :class:`.BasisTranslator`,
+        :class:`.CommutationAnalysis`, :class:`.ConsolidateBlocks`, :class:`.UnitarySynthesis`,
+        and others.
+
+      * Improvements to the circuit library that increase compilation quality and
+        construction speed. These changes include developments such as improved qubit
+        tracking for :class:`.HighLevelSynthesis`, structural circuits as part of the library
+        such as IQP, and the addition of :class:`.PauliEvolutionGate` and its operability with 
```

```suggestion
        such as :func:`.iqp`. Additionally the :class:`.PauliEvolutionGate` has been updated to leverage high level synthesis plugins including a new plugin that leverages the circuit synthesis library 
```

I would be tempted to add a sphinx ref to the docs section on high level synthesis plugins so there is a link to the details.